src/components/explosions/Explosion.module.scss

```diff
@@ -1,20 +1,17 @@
 .Explosion {
+  --explosion-bg-x: calc(0px - var(--bg-double-block-size) * 16);
+  --explosion-bg-y: calc(0px - var(--bg-double-block-size) * 8);
+
   width: var(--double-block-size);
   height: var(--double-block-size);
-  background: center / contain;
-  animation: explosion 2s forwards;
+  background: black url("../../assets/images/sprite-big.png") no-repeat
+    /* background-position */ var(--explosion-bg-x) var(--explosion-bg-y);
```

Comment:
do we need this comment here?

Reply:
Not really. That's actually for readability.

Reply:
I think `background-position` property would fit here beter.

Reply:
That way you're adding to separate properties, and the browser adds some position first and then overrides it later. That's why smart editors suggest not doing it.